docs/examples/irf_tool_config.json

```diff
@@ -12,7 +12,9 @@
   },
   "DL3FixedCuts": {
     "fixed_gh_cut": 0.7,
+    "fixed_gh_efficiency": 0.9,
```

Comment:
Same as above in the other config.

Reply:
Just an arbitrary choice again. Would you prefer some other value?

Reply:
I was not talking about the value, but the name of the configuration item as in the comment above on the `DL3FixedCuts`

Reply:
Ok